<ui version="4.0" >
 <class>NewPropertyDialog</class>
 <widget class="QDialog" name="NewPropertyDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New property</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="keyLabel" >
     <property name="text" >
      <string>Property name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="keyLineEdit" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="tyoeLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="valueLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Value:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QRadioButton" name="boolTypeButton" >
     <property name="text" >
      <string>Boolean</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QCheckBox" name="boolCheckBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>True/False</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QRadioButton" name="doubleTypeButton" >
     <property name="text" >
      <string>Double</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QDoubleSpinBox" name="doubleSpinBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QRadioButton" name="intTypeButton" >
     <property name="text" >
      <string>Integer</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QSpinBox" name="intSpinBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QRadioButton" name="strTypeButton" >
     <property name="text" >
      <string>String</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QLineEdit" name="strLineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QRadioButton" name="tupleTypeButton" >
     <property name="text" >
      <string>Tuple (string list)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QLineEdit" name="tupleLineEdit" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewPropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
